src/status/TransactionClaimed.tsx

```diff
@@ -59,6 +60,15 @@ const TransactionClaimed = () => {
                     })}
                 </p>
                 <hr />
+                <BlockExplorer
+                    asset={
+                        swap().claimTx ? swap().assetReceive : swap().assetSend
+                    }
+                    txId={swap().claimTx || swap().lockupTx}
+                    address={
+                        swap().claimAddress || (swap() as SubmarineSwap).address
+                    }
+                />
```

Comment:
This makes no sense either. Why would we ever want to show `assetReceive` after we broadcasted a claim?

Reply:
my thinking was, when doing a reverse or chainswap and i successfully receive lbtc i want to open that claimTx on the success page

Reply:
Nevermind, I meant `assetSend`. Showing the send one makes no sense